tensorboard/plugins/projector/vz_projector/projectorScatterPlotAdapter.ts

```diff
@@ -604,7 +605,10 @@ export class ProjectorScatterPlotAdapter {
           colors[dst++] = c.b;
         }
       } else {
-        if (legendPointColorer != null) {
+        if (
+          legendPointColorer != null &&
+          legendPointColorer(ds, 0) != POINT_COLOR_MISSING
```

Comment:
Maybe not just the index 0. We want to check if all colors in the data set are missing, which is equivalent to colorOption null or map null.
I start to think adding null check and add return type in line 468 might make more sense
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/plugins/projector/vz_projector/vz-projector.ts;l=468?q=tensorboard%2Fplugins%2Fprojector%2Fvz_projector%2Fvz-projector.ts&ss=tensorflow%2Ftensorboard

Reply:
Updated based on some of our offline conversation.